Tools/Pipeline/Windows/ContentIcons.cs

```diff
@@ -25,6 +25,7 @@ public class ContentIcons: IDisposable
         public ContentIcons()
         {
             Icons = new ImageList();
+            Icons.ColorDepth = ColorDepth.Depth32Bit;
```

Comment:
The default value is always `Depth8Bit` both when you run under x86 or x64.
Perhaps I was compiling with x86 (=edit & continue) when I was testing that, or perhaps all my associations where good at the time, Who knows... 
I double check with the VM machine (although I updated to 1511 since then) and the latest develop build and as you can see not all icons have this problem. 
![pipelinewin10vm](https://cloud.githubusercontent.com/assets/3018589/11199732/0c9ac758-8cd9-11e5-934f-e8ba16ddd93c.png)